lib/mix/tasks/coh.context.ex

```diff
@@ -0,0 +1,187 @@
+defmodule Mix.Tasks.Coh.Context do
+  @shortdoc "Generates a context with functions around an Ecto schema representing a User"
+
+  @moduledoc """
+  Generates a context with functions around an Ecto schema representing a User.
+      mix coh.context Accounts name:string email:string
+  """
+  
+  # Code shamelessly stolen from the Phx default context generator with minor changes
+
+  use Mix.Task
+
+  alias Mix.Phoenix.{Context}
+  alias Mix.Tasks.Phx.Gen
+
+  @switches [binary_id: :boolean, table: :string, web: :string,
```

Comment:
`binary_id` option is not needed. You would determine if `:binary_id` should be used by running:

```
iex(12)> Coherence.Config.user_schema.__schema__(:type, Coherence.Config.user_schema.__schema__(:primary_key) |> hd)
:binary_id
```